Actions/AL-Go-Helper.ps1

```diff
@@ -185,6 +185,21 @@ function OutputWarning {
     }
 }
 
+function OutputDeprecationWarning {
+    Param(
+        [string] $message
```

Comment:
NIT - we could have a message and a deprecationTag - and then if the deprecationTag is set we will add "See https://aka.ms/ALGoDeprecations#<deprecationTag> for more information." to the message.

Reply:
Similarly, we can add the date as well.

E.g.
```
After <date> this warning will become an error.
```
or
```
After <date> this setting will not be used by AL-Go and it could lead to build breaks or unwanted behavior.
```
depending on what way we want to go.